pom.xml

```diff
@@ -66,4 +94,18 @@
       <url>https://repo.jenkins-ci.org/public/</url>
     </pluginRepository>
   </pluginRepositories>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <includes>
+            <include>**/JfrRootActionTest.java</include>
+          </includes>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
This config should be already specified by the parent Pom. Remove it as it is unnecessary and introduces maintenance burden.

Reply:
This configuration is still present in the latest version. Remove all the config in the <build> tags